sorting/result1.txt

```diff
@@ -0,0 +1,6 @@
+0, 2, 6
```

Comment:
It's good practice to not check in result files **unless** you use them in a test-case to assert result correctness.